storage/innobase/include/ddl0impl-file-reader.h

```diff
@@ -132,12 +136,13 @@ struct File_reader : private ut::Non_copyable {
   /** Offset to read. */
   os_offset_t m_offset{};
 
+  os_offset_t m_len{};
+
+  byte *get_io_buffer_end() { return m_io_buffer.first + m_len; }
+
   /** Pointer current offset within file buffer. */
   const byte *m_ptr{};
 
-  /** File buffer bounds. */
-  Bounds m_bounds{};
```

Comment:
Most probably `using Bounds = std::pair<const byte *, const byte *>;` can be removed as well